### PR TITLE
Test framework -  Styling Test Results in Student Interface

### PR DESCRIPTION
**Pull request by jeremygoh**
- More styling of Student Interface view of test results
- Tried a few different things but opted to go simple and keep consistency with existing core styles

screen shot 2015-12-06 at 2 30 02 pm

Depends on #2371

### CODE REVIEW

File: db/migrate/20151129233225_add_timestamps_to_test_result.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class AddTimestampsToTestResult < ActiveRecord::Migration
+  def change
+    change_table :test_results do |t|
```
Comment (jeremygoh) [inline db/migrate/20151129233225_add_timestamps_to_test_result.rb]: Ignoring this comment as in #2371

File: app/views/automated_tests/_test_script_result.html.erb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<div class='test-result'>
+  <% test_script_output = JSON.parse(test_script_result.actual_output) %>
+  <% test_script_name =
+  test_script_output['testrun']['test_script']['script_name'] %>
```
Comment (david-yz-liu) [inline app/views/automated_tests/_test_script_result.html.erb]: When possible I think it looks better to access the model objects. Something like `test_script_result.test_script.script_name`?

File: app/views/automated_tests/_test_script_result.html.erb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+  <h3><%= "#{t('automated_tests.test_run')} - #{test_script_name}" %>
+    <span class='right'>
+      <%= l(test_script_result.created_at, format: :long)%>
```
Comment (david-yz-liu) [inline app/views/automated_tests/_test_script_result.html.erb]: Put a space before the `%>`

File: app/views/automated_tests/_test_script_result.html.erb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        <% tests.each do |test| %>
+          <tr class="<%= test['status'] == 'pass' ?
+          'test-result-success' : 'test-result-failure' %>">
+            <td><%= test['name'] %></td>
+            <td><%= test['input'] %></td>
+            <td><%= test['actual'] %></td>
+            <td><%= test['expected'] %></td>
+            <td><%= test['marks_earned'] %></td>
+            <td><%= raw test['status'] == 'pass' ?
+            '&#10004;' : '&#10008;' %></td>
```
Comment (david-yz-liu) [inline app/views/automated_tests/_test_script_result.html.erb]: Use strings here liked "Passed" or "Failed"

**Comment by jeremygoh** [comment]
Updated screenshot

screen shot 2015-12-07 at 11 49 10 am

**Pull request merged**